bootstrap_cfn/stacks/ec2.json

```diff
@@ -8,11 +8,26 @@
         "SecurityGroupIngress": []
     }
   },
+  "SaltSG": {
+    "Type": "AWS::EC2::SecurityGroup",
+    "Properties": {
+        "VpcId" : { "Ref" : "VPC" },
+        "GroupDescription": "Salt Comms Security Group",
+        "SecurityGroupIngress": [
+                                    {
+                                        "IpProtocol": "tcp",
+                                        "FromPort": 4505,
+                                        "ToPort": 4506,
```

Comment:
0.0.0.0/0 is an example - it's probably 10.0.0.0/16 - which is:

{ "Fn::FindInMap" : [ "SubnetConfig", "VPC", "CIDR" ]}